### ISSUE TITLE
I don't receive the random number for admin login

### ISSUE THREAD

**Issue by NinovdVinden** [thirtybees/thirtybees#1193]
Hi,

I tried to install thirtybees on my laptop the same way I did on my computer.
After the installation when I click on "back office" it says page not found.
On my computer it says "delete install folder" and it shows me the random number behind the admin in the URL. 

Can somebody help me?
Maybe tell me if I can find that random number somewhere else in the files or database?

**Comment by therampagerado** [comment]
If you want to use this installation (or any other and the same bug appears) you can simply put a random number there. This folder's name is not written anywhere in the table so it does not matter what string it contains (but you have to remember it or have some sort of access to see it if you forget it).

Never seen this bug in my instances.

**Comment by NinovdVinden** [comment]
When I install it on my computer it provides me with that random login to access the admin login page. On my laptop that doesnt happen at all and when I try to go to the admin page it says that the page isnt available, probably because I don’t have that random number. But you’re saying I can put any random number behind admin in the URL? Because that isn’t working aswel.

**Comment by therampagerado** [comment]
Yes, this link is for novice merchants only. Anybody that installs TB for second time should be familiar with admin folder and the need to hide it with this string.

What do you mean by 'not working'? The link when installing is not working or you can't access www.website.com/admin1359135813751397 and enter your BO?

**Comment by NinovdVinden** [comment]
When I try to access www.website.com/admin[random number] it redirects me to the index. I’ll try to explain it one more time. I tried to install thirtybees on my laptop but it didn’t provide me the random number behind the admin to access the login page. When I tried it on my computer it did provide me the random number.

**Comment by therampagerado** [comment]
If the string is wrong or missing you should get 404 not be redirected to index, so probably it's some .htaccess issue.

On your laptop you probably use XAMPP or something different that is causing the issue?

**Comment by NinovdVinden** [comment]
On my computer I use wamp, that works fine. On my laptop I use Valet Plus. The thing is, on my laptop the installation is missing the part that gives me that random number.

**Comment by therampagerado** [comment]
And where's the problem on TBs part? :)


So now if you manually amend the string can you access it with the new string in place?

**Comment by NinovdVinden** [comment]
The problem is that on one device the installation provides me with the random number and on the other device the installation doesnt show that number. How many characters should the string be? Because I tried some random things and it ain’t working because I think I need that number that is provided by you in the installation.

**Comment by NinovdVinden** [comment]
> If the string is wrong or missing you should get 404 not be redirected to index, so probably it's some .htaccess issue.
> 
> On your laptop you probably use XAMPP or something different that is causing the issue?

Well, when I only type admin in the URL it shows me a 404. When I put admin with a random string behind it, it redirects to index.

**Comment by Traumflug** [comment]
FWIW, this random string behind `admin...` gets generated when accessing `admin/` for the first time. Usually by clicking 'View Backoffice' on the last installer page.

If you forgot what the random string is, use FTP (remote server) or a file browser (local installation) to look it up.

**Comment by NinovdVinden** [comment]
I know that. On my computer it’s generated after the installation but on my laptop not. It’s both a new installation. And in what file should that number be in?



Op 24 mei 2020 om 22:03 heeft Traumflug <notifications@github.com> het volgende geschreven:

﻿

FWIW, this random string behind admin... gets generated when accessing admin/ for the first time. Usually by clicking 'View Backoffice' on the last installer page.

If you forgot what the random string is, use FTP (remote server) or a file browser (local installation) to look it up.

**Comment by Traumflug** [comment]
This number isn't stored anywhere, it gets generated on the fly.

To generate one, just access `admin/` and it'll get generated. After that you access back office by the new name, of course.

If this still doesn't work, rename the folder manually. `admin` plus some random characters. There's no secret behind all this, it's just renaming a folder to an unguessable name.

**Comment by NinovdVinden** [comment]
Okay thanks, but then it says page not available.

**Comment by Traumflug** [comment]
Is this _Valet Plus_ you talk about this: https://www.linksys.com/us/support-product?pid=01t80000003K7ZWAA0, a WLAN router? Not exactly likely this has the capabilities of a full-fledged web server.

**Comment by NinovdVinden** [comment]
No not at all. I’m talking about this: https://github.com/weprovide/valet-plus

**Comment by Traumflug** [comment]
Same for this. It's a development environment, not a web server.

Actually, a bit better than the router. You can use this environment to find out what's going on.

**Comment by NinovdVinden** [comment]
It is a web server, it runs apache.
Besides that, I tried to change the folder name but I still get page is not available.
This is the redirect link: http://thirty.test/index.php?controller=AdminLogin&token=b4e6c599029ccf7713216b8237d78e44

**Comment by NinovdVinden** [comment]
On my pc I can change the folder name to whatever I want and it keeps working but when I try it on my laptop the page isnt available, really frustrating.

**Comment by NinovdVinden** [comment]
This is what i get.
![0BF5AE90-5773-43AD-9BCE-72B78F7DC375](https://user-images.githubusercontent.com/15921588/82766290-2bd7ec80-9e1e-11ea-9c0f-67181b952ee9.jpeg)

**Comment by Jorgemgm** [comment]
See your installation folder, it should be like (the key is admin and install folders):
Adapter/
**admin/**
cache/
classes/
CODE_OF_CONDUCT.md
composer.json
config/
…
**install/**
…. 

1º: Delete “install” or rename the “install” folder (WITHOUT deleted or renamed the “install” folder, you can’t access to the backoffice )

2º Check if your admin is only admin or are like “adminfdsafsrsdfsd”

3º Go to the browser and put: “thirty.test/admin/” or “thirty.test/adminfdsafsrsdfsd/”

4º: If you want you can rename “admin” or “adminfdsafsrsdfsd” for what you want, like “admin123456789”

5º Go to the browser and put: “thirty.test/admin123456789/”

**Comment by NinovdVinden** [comment]
Thanks for taking your time into this question of mine. I did all those steps already. That’s what I’m trying to say. Originally it says “admin” and when I go to te admin page it says page not avalaible (see the picture I sent before). If I change the foldername to admin4728u2 for example and I go to that URL it does the exact same thing. I don’t know why this is happening because on my computer it works just fine. The thing is: on my computer during the installation it told me my random number. On my laptop during the installation this step wasnt shown to me. I tried to install it again but it still wasnt happening. Then I did what you told me to do so I changed the foldername but it still isnt working unfortunately.
